code/__defines/items_clothing.dm

```diff
@@ -148,7 +148,7 @@
 #define BODYTEMP_HEATING_MAX            30  // The maximum number of degrees that your body can heat up in 1 tick,   when in a hot  area.
 
 #define BODYTEMP_HEAT_DAMAGE_LIMIT 360.15 // The limit the human body can take before it starts taking damage from heat.
-#define BODYTEMP_COLD_DAMAGE_LIMIT 260.15 // The limit the human body can take before it starts taking damage from coldness.
+#define BODYTEMP_COLD_DAMAGE_LIMIT 273.15 // The limit the human body can take before it starts taking damage from coldness.
```

Comment:
I'm not sure this change is really necessary. 0 degrees celsius shouldn't start inflicting damage on somebody, especially when compared to the heat threshold.